htdocs/js/apps/DragNDrop/dragndrop.css

```
@@ -0,0 +1,58 @@
+.dd-container {
+	width: 350px;
+	float: left;
+	margin: 10px !important;
+	padding: 0;
+	color: #000000;
+	border: 1px solid #388E8E !important;
+	border-radius: 5px;
+	text-align: center;
+}
+.nestable-label {
```

It would be good to add a space between the successive rules.  For example:
```suggestion
}

.nestable-label {
```
This should be done throughout this file.  This is a rather standard formatting guideline for css.